resources/etc/init.d/openhab2

```diff
@@ -130,7 +125,7 @@ EXISTINGPID=$(findpid)
 # Kill/Wait pid
 killwaitpid() {
   kpid=$1
-  echo "Waiting for openHAB to stop..."
+  printf "Waiting for openHAB to stop..."
```

Comment:
Cosmetics: The following echo is a bit unclear. How about `echo "\nTimeout - Killing the process with pid $kpid..."`, after `done` add an `echo " OK"`